ganttproject/src/net/sourceforge/ganttproject/task/algorithm/RecalculateTaskCompletionPercentageAlgorithm.java

```diff
@@ -43,29 +43,52 @@ public void run(Task task) {
     recalculateSupertaskCompletionPercentageBottomUp(task, facade);
   }
 
+  private class PropagatedDays {
+    public int myCompletedDays;
+    public long myPlannedDays;
```

Comment:
Why planned is long while completed is int? In the code, completed is like 100x planned so it should be at least wider